geps/gep-3171/index.md

```diff
@@ -1,7 +1,7 @@
 # GEP-3171: Percentage-based Request Mirroring
 
 * Issue: [#3171](https://github.com/kubernetes-sigs/gateway-api/issues/3171)
-* Status: **Experimental**
+* Status: Standard
```

Comment:
Recommend keeping formatting
```suggestion
* Status: **Standard**
```

Reply:
Why? all of other index.md files are without **

Reply:
More importantly, the metadata.yaml file in this directory must also be updated. (That's actually the canonical location).